src/fcntl.rs

```diff
@@ -69,6 +71,8 @@ pub fn fcntl(fd: RawFd, arg: FcntlArg) -> Result<c_int> {
             F_ADD_SEALS(flag) => libc::fcntl(fd, ffi::F_ADD_SEALS, flag.bits()),
             #[cfg(target_os = "linux")]
             F_GET_SEALS => libc::fcntl(fd, ffi::F_GET_SEALS),
+            #[cfg(target_os = "darwin")]
+            F_FULLFSYNC => libc::fcntl(fd, ffi::F_FULLFSYNC),
```

Comment:
Sorry, I was confused because of https://github.com/untitaker/nix/blob/69c5d686ae5c64ea758ceaab482e5a3789397ee2/nix-test/build.rs#L10